src/logging.cpp

```diff
@@ -175,28 +173,23 @@ static const std::map<std::string, BCLog::LogFlags> LOG_CATEGORIES_BY_STR{
     {"txreconciliation", BCLog::TXRECONCILIATION},
     {"scan", BCLog::SCAN},
     {"txpackages", BCLog::TXPACKAGES},
-    {"1", BCLog::ALL},
-    {"all", BCLog::ALL},
 };
 
 static const std::unordered_map<BCLog::LogFlags, std::string> LOG_CATEGORIES_BY_FLAG{
     // Swap keys and values from LOG_CATEGORIES_BY_STR.
     [](const std::map<std::string, BCLog::LogFlags>& in) {
         std::unordered_map<BCLog::LogFlags, std::string> out;
```

Comment:
In commit "logging, refactor: make category special cases explicit" (eacf21f7cb3d4122fd196a2e49b2d7857a611bcb)
Although you're not touching this line, perhaps as long as you're in the vicinity, add:
```
        out.reserve(in.size());
```

Reply:
I try to avoid doing unrelated changes to surrounding lines because that makes the diff harder to review.

Reply:
Good point, and now that I look at it again, this is insanely unimportant for performance, because this happens once during startup!